components/bugsnag/actions/update-error-severity/update-error-severity.mjs

```diff
@@ -0,0 +1,57 @@
+import bugsnag from "../../bugsnag.app.mjs";
+import constants from "../../common/constants.mjs";
+
+export default {
+  key: "bugsnag-update-error-severity",
+  name: "Update Error Severity",
+  description: "Update an the severity status of an error in Bugsnag. [See the documentation](https://bugsnagapiv2.docs.apiary.io/#reference/errors/errors/update-an-error)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    bugsnag,
+    organizationId: {
+      propDefinition: [
+        bugsnag,
+        "organizationId",
+      ],
+    },
+    projectId: {
+      propDefinition: [
+        bugsnag,
+        "projectId",
+        (c) => ({
+          organizationId: c.organizationId,
+        }),
+      ],
+    },
+    errorId: {
+      propDefinition: [
+        bugsnag,
+        "errorId",
+        (c) => ({
+          projectId: c.projectId,
+        }),
+      ],
+    },
+    severity: {
+      type: "string",
+      label: "Severity",
+      description: "The Error's new severity. This will be reflected in the Error's overridden_severity property. This is only applicable if the override_severity option is provided.",
```

Comment:
Since we always pass 'override_severity', I think the last sentence in this description could be removed
```suggestion
      description: "The Error's new severity. This will be reflected in the Error's `overridden_severity` property",
```